pkg/controllers/cassandra/cluster_control.go

```diff
@@ -51,6 +58,17 @@ func (e *defaultCassandraClusterControl) Sync(c *v1alpha1.CassandraCluster) erro
 		)
 		return err
 	}
+	err = e.nodepoolControl.Sync(c)
+	if err != nil {
+		e.recorder.Eventf(
+			c,
+			"cassandra.defaultCassandraClusterControl",
+			ErrorSync,
+			MessageErrorSyncNodePools,
+			c,
```

Comment:
`c` is a pointer to a structure, not a string. This will print badly.

Reply:
Done.